src/db/mod.rs

```diff
@@ -30,4 +30,12 @@ impl Db {
             blocklist_cache: Arc::new(Mutex::new(None)),
         })
     }
+
+    pub async fn run_migrations(&self) -> Result<()> {
+        sqlx::migrate!("./migrations")
```

Comment:
migrate!() defaults to ./migrations

Reply:
I'd rather be explicit, honestly

Reply:
doesn't hurt to explicitly state it, otherwise it'll feel magic to people that haven't read the docs